packages/truffle-core/lib/commands/debug.js

```diff
@@ -730,6 +771,7 @@ var command = {
                 break;
               case "?":
                 printWatchExpressions();
+                printBreakpoints();
```

Comment:
For future PRs: it's helpful to upload a screenshot of output changes so reviewers don't have to grab the branch locally just to see relevant info. Looks good here though.

Reply:
Oh, I see!  Didn't realize I could do that.  Noted.